clearblade.md

```diff
@@ -1388,6 +1388,118 @@ Returns the current topics for a system
     msg.getAndDeleteMessageHistory("TestTopic", 0, null, null, null, callback); // get and delete all messages for "TestTopic"
 ~~~
 
+## messaging.setTimeout(timeout, topic, data, callback)
+
+This method publishes data to a topic after timing out.
+
+Callback setTimeout:
+
+ * @param {integer} timeout - Timeout in miliseconds
+ * @param {string} topic - String that signifies which topic to search
+ * @param {string} data - response -messages
```

Comment:
data - data to publish to the topic once it timeouts.
it's not response/messages

Reply:
Fixed